proxstar/tasks.py

```diff
@@ -150,7 +151,11 @@ def process_expiring_vms_task():
                             vm.name, vm.id
                         )
                     )
-                    # send_stop_ssh_tunnel(vm.id) # TODO (willnilges): Remove target from targets file
+                    vmid = vm['vmid'] 
+                    vnc_token_key = f'vnc_token|{vmid}'
+                    vnc_token = redis_conn.get(vnc_token_key).decode('utf-8')
+                    delete_vnc_target(token=vnc_token)
+                    redis_conn.delete(vnc_token_key)
```

Comment:
This is probably unnecessary, to be honest, and would cause more problems than it's worth.